gc/base/standard/Scavenger.cpp

```diff
@@ -1737,7 +1737,11 @@ MM_Scavenger::copy(MM_EnvironmentStandard *env, MM_ForwardedHeader* forwardedHea
 			forwardedHeader->commenceFixup(destinationObjectPtr);
 
 			/* Object model specific fixup, like age */
-			_extensions->objectModel.fixupForwardedObject(forwardedHeader, destinationObjectPtr, objectAge);
+			if (_extensions->objectModel.isIndexable(destinationObjectPtr)) {
```

Comment:
Even if we end up having specialized mixed/indexable APIs (what is still up to a debate), I'd still keep the original/generic fixupForwardedObject API too (that would query isIndexable and call specialized variants). 

Bottom line, this code (an another similar one later in this file) should not change.